test/scale/scale_test_dynamic.yaml

```diff
@@ -0,0 +1,75 @@
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: scale.sc.azurelustre.csi.azure.com
+provisioner: azurelustre.csi.azure.com
+reclaimPolicy: Delete
+parameters:
+  mgs-ip-address: "172.17.0.7"
+  fs-name: "lustrefs"
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: scale-test-service
+  labels:
+    app: scale-test-service
+spec:
+  ports:
+    - port: 999
+      name: fake
+  clusterIP: None
+  selector:
+    app: csi-scale-test
+
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: scale-test-set
+spec:
+  replicas: 5
+  selector:
+    matchLabels:
+      app: csi-scale-test
+  serviceName: ""
+  template:
+    metadata:
+      labels:
+        app: csi-scale-test
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: aks-agentpool-19553938-vmss000000
+      containers:
+        - image: busybox
+          name: lustre-echo-date
+          command:
+            - "/bin/sh"
+            - "-c"
+            - while true; do echo $(date) >> /mnt/lustre/outfile_$(POD_NAME); sleep 10; done
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          volumeMounts:
+            - name: scale-pvc-lustre
+              mountPath: "/mnt/lustre"
+          resources:
+            requests:
+              memory: 100M
+              cpu: 10m
+            limits:
+              memory: 100M
+              cpu: 10m
+  volumeClaimTemplates:
+    - metadata:
+        name: scale-pvc-lustre
+      spec:
+        accessModes: ["ReadWriteMany"]
+        storageClassName: scale.sc.azurelustre.csi.azure.com
```

Comment:
this is storage class name instead of driver name, could be shorter, e.g. `azurelustre`

Reply:
Thanks for your time on reviewing our Lustre PRs :)